src/annotations/AnnotationManager.php

```diff
@@ -299,6 +306,22 @@ protected function getAnnotations($class_name, $member_type = self::MEMBER_CLASS
                 }
             }
 
+            if ($this->_traitsSupported && $member_name) {
+                $traitAnnotations = array();
+
+                if (isset($file->traitMethodOverrides[$class_name][$member_name])) {
+                    list($traitName, $originalMemberName) = $file->traitMethodOverrides[$class_name][$member_name];
+                    $traitAnnotations = $this->getAnnotations($traitName, $member_type, $originalMemberName);
+                } else {
+                    foreach ($reflection->getTraitNames() as $traitName)
+                    {
+                        $traitAnnotations += $this->getAnnotations($traitName, $member_type, $member_name);
```

Comment:
Re (1): the semantics of getAnnotations are such that it will successfully return an empty array if that member does not exist. Checking for parent annotations currently relies on this behavior. Can still address if you'd prefer, though.

Re(2): We shouldn't need to handle this case specifically, as PHP will throw a fatal error if two traits define the same method and `insteadof` isn't used to resolve the conflict.